src/Symfony/Component/Serializer/Normalizer/DateIntervalNormalizer.php

```diff
@@ -114,8 +113,10 @@ public function denormalize($data, string $type, string $format = null, array $c
             }
 
             return new \DateInterval($data);
+        } catch (NotNormalizableValueException $e) {
+            throw $e;
```

Comment:
The `NotNormalizableValueException` is never thrown inside the try block, so this is not needed.